.github/workflows/release-napi.yml

```
@@ -42,7 +42,7 @@ jobs:
 
   build:
     needs: check
-    if: needs.check.outputs.version_changed == 'true'
+    # if: needs.check.outputs.version_changed == 'true'
```

It's only for debugging on the PR.

```suggestion
    if: needs.check.outputs.version_changed == 'true'
```